cmd2/cmd2.py

```diff
@@ -412,6 +396,23 @@ def __init__(self, completekey: str = 'tab', stdin=None, stdout=None, *,
         # If False, then complete() will sort the matches using self.default_sort_key before they are displayed.
         self.matches_sorted = False
 
+        # Load modular commands
```

Comment:
I recommend adding a big comment that all attributes must be declared above this point or something like that

Reply:
Done